ow_plexil/src/plans/FaultHandlingPattern1.plp

```diff
@@ -18,14 +18,25 @@ FaultHandlingPattern1:
 
   LibraryCall InitializeAntennaAndArm ();
```

Comment:
This initialization takes a while, is not always all needed, and it's hard to tell when the "pattern" starts.  If one injects faults during the initialization, behavior is unspecified.

I suggest doing away with the `InitializeAntennaAndArm` plan, and just have each fault pattern plan perform the initialization it needs, if any.  The initializing segment should be enclosed in messages, so that one knows when the actual pattern has started.